resources/assets/icons/status/error.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><g fill="none" stroke="currentColor" stroke-width="2"><circle stroke-miterlimit="10" cx="10" cy="10" r="9"/><path stroke-linecap="round" stroke-linejoin="round" d="M7 13l6-6m0 6L7 7"/></g></svg>
```

Comment:
identical to 
![image](https://user-images.githubusercontent.com/35610748/128482306-19f5a241-0fe6-450e-b42d-f90a39865799.png)